animations/transition_widgets.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: "Flutter transition animations"
+permalink: /animations/transitions_widgets.html
+---
+
+* TOC Placeholder
+{:toc}
+
+Transition animations implement the abstract class `AnimatedWidget`. The `AnimatedWidget` class allows you to separate the widget code from the animation code in the `setState()` call. The `AnimatedWidget` class doesn’t need to maintain a `State` object to hold the animation. Use the  `AnimatedWidget` helper class (instead of `addListener()` and `setState()`) to create a reusable transition animation widget.
```

Comment:
This page begins conceptually. It's a bit much to get one's head around. Start with an English description of a Transition. Mention that they are special objects that aren't part of the animation library, etc.